config.default.yaml

```diff
@@ -704,7 +704,7 @@ sector:
 solving:
   options:
     formulation: kirchhoff
-    load_shedding: true
+    load_shedding: 100000 # Set to "false" or willingness to pay in €/MWh, e.g. 100000 €/MWh (intersect between macroeconomic and surveybased willingness to pay http://journal.frontiersin.org/article/10.3389/fenrg.2015.00055/full)
```

Comment:
In the config file, personally I believe having the parameter in €/kWh that is more readable [easy to miss zeros with €/MWh], what do you think?